server/src/main/java/org/opentosca/toscana/plugins/kubernetes/docker/mapper/MapperConstants.java

```diff
@@ -12,7 +12,7 @@
 public class MapperConstants {
     public static final String DEFAULT_IMAGE_PATH = "library/ubuntu:latest";
     public static final String DEFAULT_IMAGE_DISTRO = "ubuntu";
-    public static final String DOCKER_HUB_URL = "https://registry.hub.docker.com";
+    public static final String DOCKER_HUB_URL = "";
```

Comment:
You intend to change the default value of the platform property to an empty string, isn't it? This feels hacky. You might rather assign an empty String directly as the platform property default value.